nob.h

```diff
@@ -2513,6 +2528,9 @@ NOBDEF int closedir(DIR *dirp)
 /*
    Revision history:
 
+     1.27.0 (2025-12-30) Add .dot_reset option to cmd_run (by @Israel77)
+                         Fix support for FreeBSD (by @qundefine)
```

Comment:
Sorry to nitpick, but you misspelled my username :)

```suggestion
                         Fix support for FreeBSD (by @cqundefine)
```

Reply:
Sorry. Should be fixed now.